kedro/framework/cli/starters.py

```diff
@@ -322,15 +335,15 @@ def new(  # noqa: too-many-arguments
     shutil.rmtree(tmpdir, onerror=_remove_readonly)
 
     # Obtain config, either from a file or from interactive user prompts.
-    config = _get_config(
+    extra_context = _get_extra_context(
```

Comment:
this is passed as `extra_context` to cookiecutter, so I think it's better just name it `extra_context`.